api/management/commands/acquiredata.py

```diff
@@ -18,22 +24,29 @@ def handle(self, *args, **options):
         if not os.path.exists(options['path']):
             os.makedirs(options['path'])
 
-        disease_path = os.path.join(options['path'], 'diseases.tsv')
+        files_in_data_path = ['samples.tsv', 'mutation-matrix.tsv.bz2']
+
+        for filename in files_in_data_path:
+            table_path = os.path.join(options['path'], filename)
+            if not os.path.exists(table_path):
```

Comment:
Is there a potential issue where an outdated version of the file exists and prevents downloading of the specified version?

Reply:
Good point. I assumed that had been done for a reason and didn't really think about it. I'll change this so that the first step is to destroy the data directory and recreate it, and then rip out these `if`s.